tests/test_urls.py

```diff
@@ -119,15 +121,17 @@ def test_missing_parameters_for_new_issue(self):
 
     def test_new_issue_should_not_crash(self):
         '''Checks 500 is not accepted for /issues/new POST.'''
-        data = {'problem_category': u'mobile_site_bug',
-                'username': u'',
-                'description': u'foo',
-                'submit-type': u'github-proxy-report',
-                'url': u'http://example.com',
-                'os': u'Foobar',
-                'browser': u'BarFoo'}
-        rv = self.app.post('/issues/new', data=data)
-        self.assertNotEqual(rv.status_code, 500)
+        with patch('requests.post') as github_data:
+            github_data.status_code.return_value = 500
+            data = {'problem_category': u'mobile_site_bug',
+                    'username': u'',
+                    'description': u'foo',
+                    'submit-type': u'github-proxy-report',
+                    'url': u'http://example.com',
+                    'os': u'Foobar',
+                    'browser': u'BarFoo'}
+            rv = self.app.post('/issues/new', data=data)
+            self.assertNotEqual(rv.status_code, 500)
```

Comment:
I don't think we are testing the right thing here. The original issue for this test is #1357.
The code which was returning 500 is webcompat not github. So we should not patch this one.
if the test is making a request to GitHub is that our route is not doing the right thing.

The test was added with this in mind
https://github.com/karlcow/webcompat.com/commit/8dd737eaf57c6390c4da191d41af52898a3fa619

It was merged in https://github.com/webcompat/webcompat.com/pull/1556

And indeed if I go back to the old test and I cut the network. This is working. So there is no Github request.